.circleci/config.yml

```diff
@@ -963,6 +965,33 @@ jobs:
             docker run --env BRANCH=$BRANCH --env REVISION=$REVISION -v /home/circleci/project:/magma -v /home/circleci/project/lte/gateway/configs:/etc/magma -i -t magma-mme-build:latest /bin/bash -c "cd /magma/lte/gateway;make clang_warning_oai_upload"
       - magma_slack_notify
 
+  li_agent_test:
```

Comment:
If you want this job to be executed you need to add it to a workflow 
example here: https://github.com/magma/magma/blob/master/.circleci/config.yml#L1344